cosmwasm/lib/wasmi-runtime/Xargo.toml

```diff
@@ -0,0 +1,91 @@
+[dependencies]
+alloc = {}
+
+[dependencies.sgx_types]
+path = "../incubator-teaclave-sgx-sdk/sgx_types"
+stage = 1
```

Comment:
Similarly to my comment above, this is n=probably unneeded.
also, you can rewrite this block as the following line, under the `[dependencies]` section:
```
sgx_types = { path = "../incubator-teaclave-sgx-sdk/sgx_types", stage = 1 }
```
and the other blocks below similarly

Reply:
done